DocNetwork DN-Dev-Availability-Service.postman_collection.json

```diff
@@ -112,10 +112,243 @@
 		},
 		{
 			"name": "Jira",
-			"item": [],
+			"item": [
+				{
+					"name": "Get Project issues Type",
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "username",
+									"value": "{{jira_email}}",
+									"type": "string"
+								},
+								{
+									"key": "password",
+									"value": "{{jira_api_key}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "user",
+								"value": "",
+								"type": "text",
+								"disabled": true
+							},
+							{
+								"key": "Accept",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "https://docnetwork.atlassian.net/rest/api/3/issuetype/project?projectId=10008",
```

Comment:
Remembering that it would be cool for the availability viewer project to one day be useful to organizations beyond DocNetwork, we should try to pull DN-specific artifacts out into variables.  That includes the `docnetwork.atlassian.net` server name and `projectId` here, right?

Can you do a pass of this logic to pull anything DN-specific out into variables?